packages/js/src/components/DefaultSeoDataAlert.js

```diff
@@ -0,0 +1,135 @@
+import PropTypes from "prop-types";
+import { ExclamationCircleIcon } from "@heroicons/react/solid";
+import { select, useDispatch } from "@wordpress/data";
+import { __, sprintf } from "@wordpress/i18n";
+import { Fragment, useCallback, useMemo } from "@wordpress/element";
+import { applyFilters } from "@wordpress/hooks";
+import { Slot } from "@wordpress/components";
+import { Button } from "@yoast/components";
+import { safeCreateInterpolateElement } from "../helpers/i18n";
+
+/**
+ * Renders the Default SEO Data Alert.
+ *
+ * @param {Object} isSeoDataDefault Whether SEO titles and meta descriptions are default.
+ *
+ * @returns {wp.Element} The Default SEO Data Alert.
+ */
+export default function DefaultSeoDataAlert( {
+	isSeoDataDefault,
+} ) {
+	const postType = select( "yoast-seo/editor" ).getPostType();
```

Comment:
If we don't have constant file we can add a constant at the top of the file right after the import:
```js
const STORE_NAME_EDITOR = "yoast-seo/editor";
const STORE_NAME_CORE = "core/edit-post";
```
You can use it 3-4 times in this file.